tests/v2/validation/rbac/rbac_additional_test.go

```diff
@@ -60,116 +52,131 @@ func (rb *RBACAdditionalTestSuite) SetupSuite() {
 	require.NoError(rb.T(), err)
 
-	addNewUserAsPO, err := users.CreateUserWithRole(rb.client, users.UserConfig(), standardUser)
-	require.NoError(rb.T(), err)
-	addNewUserAsPOClient, err := rb.client.AsUser(addNewUserAsPO)
+	log.Info("Adding a standard user as project Owner in the admin project")
+	errUserRole := users.AddProjectMember(rb.client, createProjectAsAdmin, newUser, roleProjectOwner, nil)
+	require.NoError(rb.T(), errUserRole)
+	standardUserClient, err = standardUserClient.ReLogin()
 	require.NoError(rb.T(), err)
+	return standardUserClient, createProjectAsAdmin
+}
 
-	errUserRole2 := users.AddProjectMember(rb.additionalUserClient, rb.standardUserCOProject, addNewUserAsPO, roleProjectOwner, nil)
-	require.NoError(rb.T(), errUserRole2)
+func (rb *RBACAdditionalTestSuite) TestProjectOwnerCanAddProjectOwner() {
+	subSession := rb.session.NewSession()
+	defer subSession.Cleanup()
 
-	addNewUserAsPOClient, err = addNewUserAsPOClient.ReLogin()
+	standardUserClient, adminProject := rb.testSetupUserAndProject()
+	rb.T().Logf("Creating an additional standard user")
+	additionalUser, additionalUserClient, err := setupUser(rb.client, standardUser)
 	require.NoError(rb.T(), err)
 
-	userGetProject, err := projects.GetProjectList(addNewUserAsPOClient, rb.cluster.ID)
+	log.Info("Validating if Project Owner can add another Project Owner")
+	errUserRole := users.AddProjectMember(standardUserClient, adminProject, additionalUser, roleProjectOwner, nil)
+	require.NoError(rb.T(), errUserRole)
+	additionalUserClient, err = additionalUserClient.ReLogin()
+	require.NoError(rb.T(), err)
+
+	userGetProject, err := projects.GetProjectList(additionalUserClient, rb.cluster.ID)
 	require.NoError(rb.T(), err)
 	assert.Equal(rb.T(), 1, len(userGetProject.Data))
-	assert.Equal(rb.T(), rb.standardUserCOProject.Name, userGetProject.Data[0].Name)
+	assert.Equal(rb.T(), adminProject.Name, userGetProject.Data[0].Name)
 
-	errRemoveMember := users.RemoveProjectMember(rb.standardUserClient, addNewUserAsPO)
+	errRemoveMember := users.RemoveProjectMember(standardUserClient, additionalUser)
 	require.NoError(rb.T(), errRemoveMember)
 
-	userProjectEmptyAfterRemoval, err := projects.GetProjectList(addNewUserAsPOClient, rb.cluster.ID)
+	userProjectEmptyAfterRemoval, err := projects.GetProjectList(additionalUserClient, rb.cluster.ID)
 	require.NoError(rb.T(), err)
 	assert.Equal(rb.T(), 0, len(userProjectEmptyAfterRemoval.Data))
-	users.RemoveProjectMember(rb.additionalUserClient, rb.additionalUser)
 }
 
-func (rb *RBACAdditionalTestSuite) ValidateCannotAddMPMsAsProjectOwner() {
-	createProjectAsCO, err := createProject(rb.client, rb.cluster.ID)
+func (rb *RBACAdditionalTestSuite) TestUserManageProjectMemberCannotAddProjectOwner() {
+	subSession := rb.session.NewSession()
+	defer subSession.Cleanup()
+
+	standardUserClient, adminProject := rb.testSetupUserAndProject()
+	rb.T().Logf("Creating an additional standard user")
```

Comment:
I think all of the logs in these tests for creating the user are redundant. Or at the least, should just be included in the called function as a log instead of every test. Thoughts?

Reply:
Unfortunately each of these tests are creating an additional user on which the tests are run. I will remove the statement anyway as the next line is descriptive enough.